### ISSUE TITLE
"url.replace is not a function"

### ISSUE THREAD

**Issue by finnoleary** [Rotonde/rotonde-client#93]
After opening console.log, it's filled with:
``` 
feed.js:168 Uncaught (in promise) TypeError: url.replace is not a function
    at to_hash (feed.js:168)
    at Entry.is_visible (entry.js:216)
    at Feed.refresh (feed.js:136)
    at Home.discover_next (home.js:158)
    at Portal.discover (portal.js:60)
    at <anonymous>
```

**Comment by neauoire** [comment]
This should be fixed now :)

**Comment by eelfroth** [comment]
this is fixed in #95 
issue can be closed
